core/src/classes/plugin.ts

```diff
@@ -111,15 +111,17 @@ export enum AggregationMethod {
 }
 
 export enum FilterOperation {
-  Equal = "equals",
```

Comment:
Just in case you haven't considered it, we have some work to do to maintain compatibility here.
* db migrations?
* deprecations for code config?
* still handle old value?

Reply:
Appreciate it.  I do have those written down as items to check for here.